src/components/Typography/Typography.stories.js

```diff
@@ -0,0 +1,134 @@
+import Typography from './Typography.vue';
+import { H1, H2, H3, P, Caption } from './TypographyAliases.vue';
```

Comment:
I pulled down this branch and Storybook's internal logic that generates documentation really didn't like this import because `TypographyAliases.vue` doesn't have a default import. So when I ran Storybook, I didn't get any Typography stories. 
![image](https://user-images.githubusercontent.com/9774690/122789682-2ea14d00-d27d-11eb-9e20-5c66089286ed.png)


I resolved it by changing these 2 lines to 
```
import { Typography, H1, H2, H3, P, Caption } from '../Typography';
```

There might be a better way, but as it's written currently the stories don't render 😔